### PR TITLE
Add spanish locale

### PR DESCRIPTION
**Pull request by vrunoa**
[work in progress]
Add spanish translations

### CODE REVIEW

**Comment by KazuCocoa** [comment]
Thanks!
Can you manage it in https://crowdin.com/translate/appium-desktop/ ?
cc https://github.com/appium/appium-desktop/blob/master/CONTRIBUTING.md#localization

**Comment by vrunoa** [comment]
@KazuCocoa  sure. I just created an account on crowdin but im getting a 404 for the project. Do you guys need to add me?

File: assets/locales/es/translation.json [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+  "Toggle Full Screen": "Alternar Pantalla Completa",
+  "Languages": "Idiomas",
+  "View": "Vista",
+  "Reload": "Recargar",
+  "Toggle Developer Tools": "Alternar Developer Tools",
+  "Learn More": "Aprender más",
+  "Undo": "Deshacer",
+  "Cloud Providers": "Proveedores en la nube",
+  "Environment Variables": "Variables de entorno",
+  "Save and Restart": "Giardar y Reiniciar",
```
Comment (jlipps) [inline assets/locales/es/translation.json]: `Guardar`?

File: assets/locales/es/translation.json [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+  "Inspect element": "Inspeccionar elemento",
+  "Session closed due to inactivity": "Sesión finalizada debido a inactividad",
+  "Session has been terminated": "Sesión finalizada",
+  "frameworkNotSupported": "Framework '{{framework}}' no soportado",
+  "stopServerFailed": "Falló al intentar parar el servidor: {{message}}",
+  "logsNotAvailableError": "Ha ocurrido un error: Logs no disponibles",
+  "findElementFailure": "Falló encontrar el elemento '{{methodName}}'. Intentar actualizar e intentar nuevamente.",
+  "Could not start session": "No se pudo iniciar sesión",
+  "couldNotConnect": "No se pudo conectar al servidor; esta seguro que esta funcionando?",
+  "sauceCredentialsRequired": "Usuaior/Access Key Sauce son necesarios!",
```
Comment (jlipps) [inline assets/locales/es/translation.json]: `Usuario`?

File: assets/locales/es/translation.json [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+  "Perfecto SecurityToken is required": "Perfecto SecurityToken es necesario!",
+  "browserstackCredentialsRequired": "Usuaior/Access Key Browserstack son necesarios!",
```
Comment (jlipps) [inline assets/locales/es/translation.json]: `Usuario`? O 'usaior' es una palabra técnica de que nunca he oído antés?

File: assets/locales/es/translation.json [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+  "bitbarCredentialsRequired": "Bitbar API key es necesaria!",
+  "kobitonCredentialsRequired": "Usuaior/API Key KOBITON son necesarios!",
```
Comment (jlipps) [inline assets/locales/es/translation.json]: `Usuario?`

File: assets/locales/es/translation.json [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+  "errorGettingPreset": "Error obteniendo los prestablecidos: {{message}}",
+  "errorDeletingPreset": "Ha ocurrido un error al intentar borrar un prestablecido: {{message}}",
+  "Stop Server": "Parar Appium",
+  "Close Logs": "Cerrar Logs",
+  "Stopping…": "Parando…",
+  "Start Inspector Session": "Start Inspector Session",
+  "Get Raw Logs": "Obtener Logs",
+  "The server is running": "Appium esta corriendo",
+  "The server is stopped": "Appium ha parado",
+  "The server is waiting for all connections to close": "Appium esta esperando que todas las conexiones cierren",
```
Comment (jlipps) [inline assets/locales/es/translation.json]: está

File: assets/locales/es/translation.json [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+  "Perform some actions to see code show up here": "Perform some actions to see code show up here",
+  "Recorder": "Recorder",
+  "Click swipe start point": "Click swipe start point",
+  "Click swipe end point": "Click swipe end point",
+  "Attribute": "Atributo",
+  "Value": "Valor",
+  "Find By": "Buscar por",
+  "Selector": "Selector",
+  "Interactions are not available for this element": "No se puede interactuar con este elemento",
+  "usingXPathNotRecommended": "No se recomienda usar XPath, esto puede producir fragiles tests.",
```
Comment (jlipps) [inline assets/locales/es/translation.json]: frágiles

**Comment by KazuCocoa** [comment]
oops, https://crowdin.com/project/appium-desktop is correct project root. sorry for the wrong url...

File: assets/locales/es/translation.json (hunk 2 above)
Comment (vrunoa) [reply assets/locales/es/translation.json]: 🤦🏻‍♂️

File: assets/locales/es/translation.json (hunk 3 above)
Comment (vrunoa) [reply assets/locales/es/translation.json]: 🤦🏻‍♂️

**Comment by vrunoa** [comment]
@KazuCocoa added the translations on crowdin. Updates; based on @jlipps feedback, sorry for my bad spanish 😉

**Comment by mykola-mokhnach** [comment]
@vrunoa Did you try to execute the stuff locally? Perhaps, we need to rename the locale from `es-ES` to `es` in Crowdin?

**Comment by vrunoa** [comment]
@mykola-mokhnach it's fine we can go with es-ES.

**Comment by dpgraham** [comment]
Is there anything in this that wasn't set in https://crowdin.com/project/appium-desktop/es-ES#?

If there is then I'll re-open this PR. Otherwise, this will be covered by the next sync PR.

**Pull request closed without merging**